.github/workflows/test-upgrade-submodules.yml

```diff
@@ -0,0 +1,16 @@
+name: test-upgrade-submodules
+on: pull_request
+
+jobs:
+  verify_pull_request:
+    if: github.actor == 'DevRel Bot'
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Get submodules
+        run: git submodule update --init --recursive
```

Comment:
@2can I'm not sure if I understand it

What we have now:
automatic PRs that already update submodules

What we want:
Pipeline on *every* PR (including bot, but also manual) that builds docer

Reply:
@lkostrowski OK - updated to run on every PR

Reply:
Ensure submodules are only cloned, we can configure checkout action
https://github.com/actions/checkout

Reply:
@lkostrowski updated